app/models/analysis_metadatum.rb

```diff
@@ -268,12 +268,14 @@ def create_payload
           outputs = []
           workflows.each do |workflow|
             outs = workflow['outputs'].values
-            outs.each do |o|
-              outputs << {
-                  'name' => o.split('/').last,
-                  'file_path' => o,
-                  'format' => o.split('.').last
-              }
+            if outs.present?
```

Comment:
You could skip the if check and do `outs&.each do |o|` instead --  dealer's choice :)

Reply:
Ooooh snazzy.  Didn't know that particular shorthand.  It's not as readable but I do like the concision.